main.c

```diff
@@ -28,39 +35,90 @@ static void LED_init() {
     TRISC7 = 0;
     LATC7 = 1;      // turn the led off
 }
-const uint8_t highVal = 200;
-const uint8_t lowVal = 50;
-uint8_t byte1 = 0x00, byte2 = 0x00;
-uint8_t* b1 = &byte1, b2 = &byte2;
+static void LinAc_init() {
```

Comment:
Can you break out all the linear actuator related code into a separate pair of files (header + source).

Reply:
sure